src/glfs-cli.c

```diff
@@ -271,7 +271,7 @@ parse_options (struct cli_context *ctx)
         opterr = 0;
 
         while (true) {
-                opt = getopt_long (argc, argv, "o:", long_options,
```

Comment:
The main reason I left out `-h` as a short option for help text was to minic coreutils which does not have a short option for help.

Additionally, I wanted to keep consistency between the tools. Specifically, the `gfls` tool has a `-h` flag which means human readable.